manifest_prod.yml

```diff
@@ -1,11 +1,14 @@
 ---
-inherit: manifest_base.yml
+buildpack: staticfile_buildpack
+applications:
+  - name: proxy
+    memory: 64M
 instances: 4
 routes:
-- route: beta.fec.gov
-- route: fec-prod-proxy.app.cloud.gov
-- route: transition.fec.gov
-- route: www.fec.gov
+  - route: beta.fec.gov
```

Comment:
Ah, we haven't deprecated `beta.fec.gov` yet, have we? :-)

Reply:
@ccostino I don't think we have - looks like it still works. Maybe it's to redirect old links people saved when we were in beta?